Datum/Public/Get-FileProviderData.ps1

```diff
@@ -4,15 +4,30 @@ function Get-FileProviderData {
         $Path,
 
         [AllowNull()]
-        $DatumHandlers = @{}
+        $DatumHandlers = @{},
+
+        [switch]
+        $AsJsonString
     )
     Write-Verbose "Getting File Provider Data for Path: $Path"
     $File = Get-Item -Path $Path
     switch ($File.Extension) {
-        '.psd1' { Import-PowerShellDataFile $File           | ConvertTo-Datum -DatumHandlers $DatumHandlers }
-        '.json' { ConvertFrom-Json (Get-Content -Raw $Path) | ConvertTo-Datum -DatumHandlers $DatumHandlers }
-        '.yml'  { ConvertFrom-Yaml (Get-Content -raw $Path) -ordered | ConvertTo-Datum -DatumHandlers $DatumHandlers }
-        
-        Default { Get-Content -Raw $Path }
+        '.psd1' { $Object = Import-PowerShellDataFile $File }
+        '.json' { $Object = ConvertFrom-Json (Get-Content -Raw $Path) }
+        '.yml'  { $Object = ConvertFrom-Yaml (Get-Content -raw $Path) -ordered }
+
+        Default { $Object = Get-Content -Raw $Path }
+    }
+
+    $Object = ConvertTo-Datum -DatumHandlers $DatumHandlers -InputObject $Object
+
+    if($AsJsonString.IsPresent) {
+        Write-Output ($Object | ConvertTo-Json -Compress)
+    } else {
+        if ($o -is [array]) {
+            Write-Output $Object -NoEnumerate
```

Comment:
Maybe -NoEnumerate should be default? (note to self)

Reply:
How do you want to do this? With a global splat somewhere?